pkgs/stdenv/freebsd/linkBootstrap.sh

```diff
@@ -6,5 +6,9 @@ for path in $paths; do
         exit 1
     fi
     mkdir -p $out/$(dirname $path)
-    ln -s $src/$path $out/$path
+    if [[ -d $src/$path ]]; then
+        ln -s $src/$path $out/$path
+    else
+        cp -RL $src/$path $out/$path
```

Comment:
Why do you need to copy? If this is required to prevent a dependency on the bootstrap tarball contents I'd prefer to fix it somewhere else

Reply:
Done. I put it in the nix code instead of in the shell code so it isn't part of the derivation hash.

Reply:
Thanks, I'll do some more build tests then merge